app/assets/stylesheets/petitions/views/_home.scss

```diff
@@ -87,7 +91,15 @@
   }
 }
 
-.local-to-you h2,
-.start-petition h2 {
-  @include bold-24;
+#local-to-you-heading,
```

Comment:
Oops! Didn't mean to commit that. Will change that back.
On 8 Jul 2015 10:28, "Alexander Bobin" <notifications@github.com> wrote:

> In app/assets/stylesheets/petitions/views/_home.scss
> <https://github.com/alphagov/e-petitions/pull/292#discussion_r34130068>:
>
> > @@ -87,7 +91,15 @@
> >    }
> >  }
> >
> > -.local-to-you h2,
> > -.start-petition h2 {
> > -  @include bold-24;
> > +#local-to-you-heading,
>
> We should not use the #anchors for headings. Better to use the classes as
> was previously done.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/e-petitions/pull/292/files#r34130068>.
>